concent_api/verifier/tasks.py

```diff
@@ -36,6 +35,12 @@
 from .utils import unpack_archive
 
 
+# OpenCV is imported conditionally because it requires system module to work,
```

Comment:
`system module` -> `a system package` or `OpenCV package installed in the system`

To make it clear that you don't refer to the `sys` module or any other Python module.